src/rs.cpp

```diff
@@ -2997,7 +3003,13 @@ void rs2_update_firmware_unsigned_cpp(const rs2_device* device, const void* imag
 {
     VALIDATE_NOT_NULL(device);
     VALIDATE_NOT_NULL(image);
-    VALIDATE_FIXED_SIZE(image_size, unsigned_fw_size); // check if the given FW size matches the expected FW size
+    // check if the given FW size matches the expected FW size
+    auto fw_image_sz = { unsigned_fw_size, unsigned_sr300_size };
+    if (std::find(fw_image_sz.begin(), fw_image_sz.end(), image_size) == fw_image_sz.end())
```

Comment:
That's three times now... maybe this should be abstracted into a helper function?